lang/c++/api/DataFile.hh

```diff
@@ -102,6 +103,11 @@ public:
      */
     void syncIfNeeded();
 
+    /**
+     * Returns offset to the last sync marker written.
+     */
+    uint64_t getLastSync();
```

Comment:
I like your suggestion. I will make the change.

However, I chose this name since it matched with the terminology used in DataFileReader. DataFileReader already exposes the following: pastSync(), previousSync(), etc. which denote the same. Should we rationalize these APIs as well in a future change?